hs/t/n/view_alias_shadowed.txt

```diff
@@ -0,0 +1,8 @@
+reachc: error[RE0123]: The name `A_y` has already been used by a Participant, View or API at: ./view_alias_shadowed.rsh:5:16:application
```

Comment:
Your comment that the name clash handling is already done... I think you should have just a few more examples. This is when a View alias conflicts with an API name. What about when a View alias conflicts with another View alias, or another name?

Reply:
ok, Ill add those